glean-core/rlb/examples/delayed-ping-data.rs

```diff
@@ -127,12 +127,13 @@ fn main() {
         "accumulate_ten_and_orderly_shutdown" => {
             log::debug!("incrementing by 10, waiting, shutdown. should trigger a flush.");
             glean_metrics::sample_counter.add(10);
+            std::thread::sleep(std::time::Duration::from_millis(100));
             glean::shutdown();
         }
         "submit_ping" => {
             log::info!("submitting PrototypePing");
             PrototypePing.submit(None);
-
+            std::thread::sleep(std::time::Duration::from_millis(100));
```

Comment:
Gonna need some comments about what these sleeps are doing (ie, why they're needed here in the test, but not out in the wider world where people use the SDK)